extension-manifest-v3/src/pages/settings/views/privacy.js

```diff
@@ -180,7 +176,8 @@ export default {
           </section>
 
           <gh-settings-devtools
-            onis-shown=${onDevModeEnabled}
+            onshown="${html.set('devMode', true)}"
```

Comment:
For that one-liners you can use `html.set` helper with custom value as a second argument. No need for custom callback function here.